addon/modifiers/autofocus.js

```diff
@@ -4,22 +4,21 @@ import { next } from '@ember/runloop';
 const DEFAULT_SELECTOR =
   'input:not([disabled]):not([readonly]),textarea:not([disabled]):not([readonly])';
 
-export default modifier(function autofocus(
-  element,
-  [selector = DEFAULT_SELECTOR],
-  { disabled }
-) {
-  if (disabled) {
-    return;
-  }
+export default modifier(
+  function autofocus(element, [selector = DEFAULT_SELECTOR], { disabled }) {
+    if (disabled) {
+      return;
+    }
 
-  const childElement = element.querySelector(selector);
+    const childElement = element.querySelector(selector);
 
-  next(function () {
-    if (childElement) {
-      childElement.focus();
-    } else {
-      element.focus();
-    }
-  });
-});
+    next(function () {
+      if (childElement) {
+        childElement.focus();
+      } else {
+        element.focus();
+      }
+    });
+  },
+  { eager: false }
```

Comment:
Only added this line, the rest are Prettier changes.

Reply:
This change seems unrelated? Can we do it in another PR if that's the case please?

Reply:
Which change is unrelated? The eager option resolves the deprecation. The Prettier changes are a result of adding new code and they are necessary to make CI happy.

Reply:
All right, got it, thanks 👍